lib/webvalve/fake_service_config.rb

```diff
@@ -36,8 +48,9 @@ def strip_basic_auth(url)
       url.to_s.sub(%r(\Ahttp(s)?://[^@/]+@), 'http\1://')
     end
 
-    def service_enabled_in_env?
-      WebValve::ENABLED_VALUES.include?(ENV["#{service_name.to_s.upcase}_ENABLED"])
+    def service_enabled_in_env?(default:)
+      value = ENV.fetch("#{service_name.to_s.upcase}_ENABLED", default).to_s
+      WebValve::ENABLED_VALUES.include?(value)
```

Comment:
`ENABLED_VALUES` doesn't actually include `true` (boolean) -- only strings -- `%w(1 t true)`

Are we missing test coverage for the production env + webvalve_enabled=1 case?

Reply:
i'm converting the true boolean to a string on line 52, which is why this works. i believe we have the proper coverage.

Reply:
but i can add the non-string version for sanity.